Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue3454.cs

```diff
@@ -0,0 +1,68 @@
+﻿#if APP
+using System.Collections.Generic;
+using System.Linq;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 3454, "Picker accessibility text is wrong", PlatformAffected.macOS)]
+	public class Issue3454 : TestContentPage
+	{
+		protected override void Init()
+		{
+			var pickers = new List<Picker> {
+				new Picker(),
+				new Picker(),
+				new Picker { Title = "Title1" },
+				new Picker { Title = "Title2" },
+				new Picker { Visual = VisualMarker.Material },
+				new Picker { Visual = VisualMarker.Material },
+				new Picker { Title = "Title1", Visual = VisualMarker.Material },
+				new Picker { Title = "Title2", Visual = VisualMarker.Material },
+			};
+
+			pickers[0].SetAutomationPropertiesName("First accessibility");
```

Comment:
I noticed that with these items it doesn't read what's selected in the box. If you don't have these specified it'll read out what's selected first

- click box
- select item 6
- click off box
- click back on box (here it should read "item 6" and then the rest of the text)

Reply:
Fixed